### PR TITLE
[Tizen studio] Change screenshots for Tizen Studio 3.0

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

Take screenshots from Tizen Studio 3.0 and replace them.

### CODE REVIEW

File: docs/application/tizen-studio/web-tools/project-wizard.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Based on the selected profile and version, a list of templates is shown in the t
 
 **Figure: Selecting the profile and version**
 
-![File analysis](./media/project_wizard_profile.png)
+![File analysis](./media/create_project_wizard_version.png)
```
[the remark this answers is not in the document]
Comment (journey2w) [reply docs/application/tizen-studio/web-tools/project-wizard.md]: create_project_wizard_version.png was updated.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/452

**Comment by mijin85cho** [comment]
Thanks for the update.

**Comment by mijin80lee** [comment]
TD Sync Completed.(for Tizen_5.0_prepare)

**Pull request merged**